src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,81 @@
+---
+layout: article
+title: Keyboard
+description: Learn to efficiently and effectively navigate and access the Appwrite Console with your keyboard or the Appwriter embedded with shortcuts.
+---
+
+The Appwrite Console is optimized for using a keyboard using shortcut keys, significantly enhancing the developer's efficiency without interruption from using the mouse. These shortcut keys also benefit developers relying solely on keyboards, guaranteeing an inclusive and accessible platform. We improve developers' productivity by using the keyboard's ease of use, quick and seamless execution of essential actions, and the consistency of the shortcut key.
+
+# Accessibility  {% #accessibility %}
+
+We are committed to making Appwrite's platform accessible to all developers by following the AA-level standards of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels). Following these standards ensures that Appwrite's platform is usable by people with visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities. 
+
+Good contrast between text and backgrounds, resizable text without loss of content or functionality, and navigable interfaces via keyboard and screen readers guarantee an inclusive environment and allow developers to perform their tasks efficiently. Adhering to these guidelines ensures that Appwrite's dashboard complies with legal requirements and promotes a positive developer experience for everyone.
+
+# Shortcuts {% #shortcuts %}
+
+You can only use these keyboard shortcuts in the Appwrite Console. Appwrite's keyboard shortcuts follow the following pattern: use the first letters from the call to action and the resource, product, service, or page you're targeting.  For example, using the shortcut `G,` then `S` goes to storage, but the shortcut `G` then `E` is if you want to go to settings, because `G` then `S` is already taken, so the following letter of settings will be used for this shortcut. This pattern is used to make usage of shortcuts consistent and predictable.
```

Comment:
```suggestion
The Appwrite Console supports keyboard shortcuts that make it easier to navigate and perform common actions quicker. The shortcuts use the following pattern: use the first letter from the call to action followed by the resource, product, service, or page you're targeting.  For example, the shortcut keys `G` + `S` navigates to the project's Storage screen. Similarly, the shortcut `G` + `F` navigates to the project's Functions screen. However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. This pattern is used to make usage of shortcuts consistent and predictable.
```

Reply:
> However, there are some exceptions. The shortcut `G` + `E` navigates to project Settings, because `G` + `S` is in use for Storage. 

It would be good to reword this to explain a bit more. When there's a conflict with a shortcut key, the following letter is used. For example, since S is already for Storage, the shortcut for Settings is E.